packages/ui/src/components/GroupListItem/GroupListItemContent.tsx

```diff
@@ -27,8 +30,18 @@ export default function GroupListItemContent({
     >
       <ListItem.Icon
         fallbackText={model.title?.[0]}
-        backgroundColor={model.iconImageColor ?? undefined}
-        imageUrl={model.iconImage ?? undefined}
+        backgroundColor={
+          disableAvatars
```

Comment:
Are we applying this logic anywhere else? At some point we probably want this broken out as a helper, but doesn't have to be now.

Reply:
Same for the image url below.

Reply:
Cool, at least abstracted backgroundColor out to handle both this and Avatar. PTAL!